test/CSharpLiteWriterUnitTests/Given_an_OdcmClass_Entity_Collection_AddLinkAsync_Method.cs

```diff
@@ -0,0 +1,138 @@
+﻿
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Its.Recipes;
+using Microsoft.MockService;
+using Microsoft.MockService.Extensions.ODataV4;
+using Newtonsoft.Json.Linq;
+using Vipr.Core.CodeModel;
+using Xunit;
+
+namespace CSharpLiteWriterUnitTests
+{
+    public class Given_an_OdcmClass_Entity_Collection_AddLinkAsync_Method : EntityTestBase
```

Comment:
Fixed the test to derive from 'NavigationPropertyTestBase'. It was not easily discoverable before.